packages/client/src/components/Location/LocationSummary.test.tsx

```diff
@@ -16,6 +16,15 @@ jest.mock("react-router-dom", () => ({
   }),
 }));
 
+const monumentText = "Het gebouw is een gemeentelijk monument.";
+const noMonumentText = "Het gebouw is geen monument.";
+const nationalCityScapeText =
+  "Het gebouw ligt in een rijksbeschermd stads- of dorpsgezicht.";
+const municipalCityScapeText =
+  "Het gebouw ligt in een gemeentelijk beschermd stads- of dorpsgezicht.";
+const noCityScapeText =
+  "Het gebouw ligt niet in een beschermd stads- of dorpsgezicht.";
+
```

Comment:
I didn't do it because the text is dynamic: `Het gebouw is een ${monument.toLowerCase()}.`

But there is an option to make some part translatable. I will look into that

Reply:
I've translated this now